### PR TITLE
Update license file; with additional layout changes

### LINKED ISSUE #4714: Me: small typo on the "Open source licenses" page
Steps to reproduce:
1. Go to the Profile &gt; "App Settings" &gt; "Open source licenses" page.
2. Scroll down to the "Notices for library: photoview" section.

<strong>Result:</strong> It appears there's an extra slash where there shouldn't be one.

### PR DESCRIPTION
**Pull request by tonyr59h**
Fixes #4714 

There are [additional changes](https://cldup.com/fMYRSKqxHR.png) on [this branch](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=android+html+collapse) that I think will improve the layout but go beyond the scope of the issue.

To test:
* Check out the Open Source License page from the bottom of `App Settings`

### CODE REVIEW

**Comment by maxme** [comment]
Greenrobot Event Bus reference is missing from the file. Layout improvement is welcome, especially if it's done already.

**Comment by tonyr59h** [comment]
EventBus added and layout updated 👍

File: WordPress/src/main/assets/licenses.html [hunk 1]
```diff
@@ -17,397 +18,31 @@
...
+libraries are licensed under <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache v2</a>:</p>
 
     <ul>
-      <li>httpmime</li>
+      <li><a href="https://github.com/google/gson">gson</a>: Copyright 2002-2008, Google</li>
+      <li><a href="https://github.com/chrisbanes/PhotoView">PhotoView</a>: Copyright 2011-2012, Chris Banes</li>
+      <li><a href="https://github.com/jukka/tagsoup">TagSoup</a>: Copyright 2002-2008, John Cowan</li>
+      <li><a href="https://github.com/bauerca/drag-sort-listview">DragSortListView</a>: Copyright 2012, Carl Bauer</li>
+      <li><a href="https://github.com/Yalantis/uCrop">uCrop</a>: Copyright 2017, Yalantis</li>
+      <li><a href="https://github.com/Yalantis/uCrop">Simple Tool Tip</a>: Copyright 2016, Xizhi Zhu</li>
```
Comment (maxme) [inline WordPress/src/main/assets/licenses.html]: Wrong url here

File: WordPress/src/main/assets/licenses.html [hunk 2]
```diff
@@ -17,397 +18,31 @@
...
+      <li><a href="https://github.com/Yalantis/uCrop">okio/okhttp</a>: Copyright 2013 Square, Inc.</li>
```
Comment (maxme) [inline WordPress/src/main/assets/licenses.html]: And wrong url there

**Comment by tonyr59h** [comment]
URL's all fixed up in 15afc161deb41826e26a61fdd0083e341defdaea

**Comment by maxme** [comment]
Looks good, :shipit:

**Pull request merged**